pkg/platform/packet/packet.go

```diff
@@ -101,6 +101,16 @@ func (c *config) LoadConfig(configBody *hcl.Body, evalContext *hcl.EvalContext)
 		}
 		return hcl.Diagnostics{err}
 	}
+
+	if c.Networking != "" && c.Networking != "calico" {
+		err := &hcl.Diagnostic{
+			Severity: hcl.DiagError,
+			Summary:  "Only calico networking is supported on Packet",
```

Comment:
```suggestion
			Summary:  "Only Calico networking is supported on Packet",
```

as it's the project name 

or 

```suggestion
			Summary:  "Only 'calico' networking is supported on Packet",
```

to indicate, that this is the only valid _value_.